Workbooks/HDInsight/Overview/Overview.workbook

```diff
@@ -0,0 +1,442 @@
+        "queryType": 0,
+        "resourceType": "microsoft.insights/components"
+      },
+      "name": "parameters - 8"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "metrics_kafka_CL\r\n| where iff(\"{Clusters}\" == \"\", true, ClusterName_s in ({Clusters}))\r\n| where InstanceName_s == \"kafka-BrokerTopicMetrics-BytesInPerSec-Count\"\r\n| summarize AggregatedValue = avg(kafka_BrokerTopicMetrics_BytesInPerSec_Count_value_d) by ClusterName_s, HostName_s, bin(TimeGenerated, 1m)\r\n| summarize IncomingBytesPerSec=sum(AggregatedValue)/toint('{Units}') by TimeGenerated, ClusterName_s\r\n| where TimeGenerated < ago(3m)\r\n| order by TimeGenerated desc",
+        "size": 0,
+        "aggregation": 5,
+        "title": "Incoming data per second by cluster",
+        "timeContext": {
+          "durationMs": 0
+        },
+        "timeContextFromParameter": "TimeRange",
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces",
+        "crossComponentResources": [
+          "{Workspace}"
+        ],
+        "visualization": "timechart",
+        "tileSettings": {
+          "showBorder": false,
+          "titleContent": {
+            "columnMatch": "ClusterName_s",
+            "formatter": 1
+          },
+          "leftContent": {
+            "columnMatch": "IncomingBytesPerSec",
+            "formatter": 12,
+            "formatOptions": {
+              "palette": "auto"
+            },
+            "numberFormat": {
+              "unit": 17,
+              "options": {
+                "maximumSignificantDigits": 3,
+                "maximumFractionDigits": 2
+              }
+            }
+          }
+        }
+      },
+      "customWidth": "50",
+      "showPin": true,
+      "name": "query - 0 - Copy - Copy - Copy"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "metrics_kafka_CL\r\n| where iff(\"{Clusters}\" == \"\", true, ClusterName_s in ({Clusters}))\r\n| where InstanceName_s == \"kafka-BrokerTopicMetrics-BytesOutPerSec-Count\"\r\n| summarize AggregatedValue = avg(kafka_BrokerTopicMetrics_BytesOutPerSec_Count_value_d) by ClusterName_s, HostName_s, bin(TimeGenerated, 1m)\r\n| summarize IncomingBytesPerSec=sum(AggregatedValue)/toint('{Units}') by TimeGenerated, ClusterName_s\r\n| where TimeGenerated < ago(3m)\r\n| order by TimeGenerated desc",
+        "size": 0,
+        "aggregation": 5,
+        "title": "Outgoing data per second by cluster",
+        "timeContext": {
+          "durationMs": 0
+        },
+        "timeContextFromParameter": "TimeRange",
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces",
+        "crossComponentResources": [
+          "{Workspace}"
+        ],
+        "visualization": "linechart"
+      },
+      "customWidth": "50",
+      "showPin": true,
+      "name": "query - 0 - Copy - Copy - Copy - Copy"
```

Comment:
you might want to go into the "advanced" settings for all of the steps and edit the "Step name" field.  right now they all show up as "query", then "query - 0 - copy", etc.  most of the time users won't see these, but its useful when they have nice names

Reply:
fixed, thanks